src/utils/zoomCenterCalculate.ts

```diff
@@ -0,0 +1,20 @@
+import { IZoomShift } from "@/interfaces/zoomShift";
+
+export const centerZoom = (zoomInfo: IZoomShift) => {
+  const xShift =
```

Comment:
hi Zack and Taylor not understand this formula, can you explain it once you have time
what means yShift, xShift
where is startPoint in the canvas , I found it in console log {X: 2500, Y: 2500}
and dragPos is X=0 and Y=0, where is the initial dragPos(X =0, Y=0) in the canvas
Thanks